rows_test.go

```diff
@@ -144,7 +144,7 @@ var parseRawCellsTests = []struct {
 		Name:  "Invalid Cell",
 		Error: "Cell had type of InlineString, but the InlineString attribute was missing",
 		RawCells: []rawCell{
-			rawCell{Type: "inlineStr", InlineString: nil, Value: &inlineStr},
```

Comment:
Are these removals just tidy up? Is the object derived/implicit anyway?

Reply:
Exactly that - because it's a slice of rawCells, you don't need to specify the type of each element.

Reply:
Thats nice